src/main/haskell/kore/test/Test/Kore/Step/BaseStep.hs

```diff
@@ -75,6 +75,7 @@ test_baseStep =
                 AxiomPattern
                     { axiomPatternLeft = asPureMetaPattern (x1 PatternSort)
                     , axiomPatternRight = asPureMetaPattern (x1 PatternSort)
+                    , axiomPatternRequires = makeTruePredicate
```

Comment:
Could you add a test that uses a different predicate? See `testCase "Preserving initial condition."` for an example.

You could do the same for the Integration test, but that's less important.

Could you also add one for the UserDefined test? There is a task for changing the axiom handling code, and having a test for a non-true condition would be nice.